components/PlantForm.js

```diff
@@ -168,14 +195,16 @@ export default function PlantForm({ onCreatePlant }) {
                 id={option.id}
                 name="fertiliserSeason"
                 value={option.value}
+                defaultChecked={isEditMode ? initialData?.fertiliserSeason.includes(option.value) : null}
               />
               <label htmlFor={option.id}>{option.label}</label>
             </div>
           ))}
         </section>
 
         <div className="button">
-          <StyledButton type="submit">Create</StyledButton>
+          <StyledButton type="submit" className={isEditMode ? "save" : ""}>{isEditMode ? "Save"  : "Create"}</StyledButton>
```

Comment:
I think they told us not to use className for the projects

Reply:
You could use 
```
 <StyledButton type="submit" isEditMode }>{isEditMode ? "Save"  : "Create"}</StyledButton>
```
and for the css 
 ```
    background-color:  ${({ isEditMode }) => (isEditMode ? "your colour" : "other colour")};
```

Reply:
Hey Levin! Thank you! We fixed that ;) and now the form also closes after saving the updates